### PR TITLE
Suppress the errors

### PR DESCRIPTION
**Pull request by SQLDBAWithABeard**
## Type of Change

 - [X ] Bug fix (non-breaking change)
 - [ ] New feature (non-breaking change, adds functionality)
 - [ ] Breaking change (effects multiple commands or functionality)
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] Nunit test is included
 - [ ] Documentation
 - [ ] Build system
 
### Purpose
 

 fixes Rob's Pain at seeing the emails from PowerShell gallery about failed Pester Tests!

Seriously, I have just added the suppress attribute to the Pester tests that were causing PowerShell gallery to email dbatools owners about failing tests. This will not alter anything other than when a Pester test is run **ON THE FILE** it will ignore those rules.

This will not effect the Pester Tests for dbatools running

### Approach

Suppresses the Pester tests with [Diagnostics.CodeAnalysis.SuppressMessageAttribute

### CODE REVIEW

**Comment by niphlod** [comment]
once again psgallery shows its "green-ness" .... maybe we should add checks for tests too to avoid this happening again ? or at least modify the template ...

File: functions/Invoke-DbaWhoisActive.ps1 [hunk 1]
```diff
@@ -254,7 +254,7 @@ Similar to running sp_WhoIsActive @get_outer_command = 1, @find_block_leaders =
         foreach ($instance in $sqlinstance) {
             try {
                 Write-Message -Level Verbose -Message "Connecting to $instance"
-                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $sqlcredential
+                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $sqlcredential -MinimumVersion 9
```
Comment (potatoqualitee) [inline functions/Invoke-DbaWhoisActive.ps1]: This file has additional changes :/ I think you missed a sync

**Comment by wsmelton** [comment]
A better option would be to simply remove the test files from the PS Gallery.

File: functions/Invoke-DbaWhoisActive.ps1 (hunk 1 above)
Comment (SQLDBAWithABeard) [reply functions/Invoke-DbaWhoisActive.ps1]: Thats odd - that didnt come form me

image

Came from 926c7c253d1112f76fef0a1373a1baf777479926

How do I resolve that?

**Comment by niphlod** [comment]
as you wish, I'm going to add scriptanalyzer checks for high-severity into CI tomorrow for functions, adding for tests will be one step away.

**Comment by potatoqualitee** [comment]
Do we close this PR? 😢

**Comment by SQLDBAWithABeard** [comment]
If functions/Invoke-DbaWhoisActive.ps1 can be removed I would say put it in whilst a decision is made about which way to go forward?  At least then we wont get the emails and the Gallery will be happy

**Comment by potatoqualitee** [comment]
b's love Invoke-DbaWhoisActive, that'd be harmful :(

**Comment by SQLDBAWithABeard** [comment]
LOL - I meant from this PR

**Comment by potatoqualitee** [comment]
where do we stand, yall?

**Comment by wsmelton** [comment]
I'm still for ignoring the test folder in your CI build/deploy to PS Gallery. It is possible to exclude that directory and other files when you use VSTS to do the build.

**Comment by potatoqualitee** [comment]
i'd like that - can we do it @wsmelton ?

**Comment by wsmelton** [comment]
Will work on this after yall get back from sqlbits.

**Comment by potatoqualitee** [comment]
Lez go, where we at? The VSTS stuff?

**Comment by potatoqualitee** [comment]
K so imma close this but not delete the branch in the event we got this way instead of the other way 🌮

**Pull request closed without merging**